test/infrastructure/clusters/test-infra/karpenter-tests/pipeline-trigger-cron.yaml

```diff
@@ -58,26 +58,26 @@ data:
      apiVersion: tekton.dev/v1beta1
      kind: PipelineRun
      metadata:
-       generateName: "upgrade-suite"
+       generateName: "upgrade-periodic-"
        namespace: karpenter-tests
      spec:
        pipelineRef:
          name: upgrade
        serviceAccountName: karpenter-tests
     EOF
-    echo "CREATED PIPELINE RUN for upgrade-suite"
+    echo "CREATED PIPELINE RUN for upgrade-periodic"
     cat <<EOF | kubectl create -f -
      apiVersion: tekton.dev/v1beta1
      kind: PipelineRun
      metadata:
-       generateName: "ipv6-suite"
+       generateName: "ipv6-periodic-"
```

Comment:
similar trailing extra `-`

Reply:
If you look at the CI tests, these are missing a `-`. This addition is intentional.

Reply:
<img width="361" alt="image" src="https://user-images.githubusercontent.com/2585992/211096015-937fcdc8-a93c-43d4-b10e-178dd28dbbc3.png">